### DISCUSSION TITLE
I am being rejected in GitHub Students because my university doesn’t’t provide emails to the students

### DISCUSSION THREAD

**Discussion by decomoraes** [community/community#112355] [GitHub Education]
That's an absurd, it looks like the GitHub student is made for rich countries, only in your developed world all the universities provide an email for each student, and your system doesn't allow to attach a pdf, so I had to screenshot the PDF, what choose I have, the pdf has the verification link, I am trying many times and I'm being rejected, I think you need to see your process because it's very exclusive.

**Comment by samus-aran** [comment]
Hi @decomoraes 👋 

We do accept other forms of enrolment if schools do not distribute academic emails. Here are a few links from our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) that may help you in these situations:

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

I can see that you have now been approved after adding an academic email. 🙂 Your educational benefits will arrive within 72 hours of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
